### ISSUE TITLE
ChannelPrunedLearner with tf.keras.layers.Conv1D

### ISSUE THREAD

**Issue by Augnine** [Tencent/PocketFlow#297]
I use tf.keras.layers.Conv1D in my own network, and i have trained it with FullPrecLearner. Then when I use ChannelPrunedLearner, there are some errors :

ERROR:tensorflow:the number of real inputs of model/encoder/encode_conv1d/conv1d/Conv2D should be 1
ERROR:tensorflow:the number of weights variable of model/encoder/encode_conv1d/conv1d/Conv2D should be 1

the structure of con1d like this:
name: "model/encoder/encode_conv1d/conv1d/Conv2D"
op: "Conv2D"
input: "model/encoder/encode_conv1d/conv1d/ExpandDims"
input: "model/encoder/encode_conv1d/conv1d/ExpandDims_1"
attr {
  key: "T"
  value {
    type: DT_FLOAT
  }
}
attr {
  key: "data_format"
  value {
    s: "NHWC"
  }
}
attr {
  key: "dilations"
  value {
    list {
      i: 1
      i: 1
      i: 1
      i: 1
    }
  }
}
attr {
  key: "padding"
  value {
    s: "VALID"
  }
}
attr {
  key: "strides"
  value {
    list {
      i: 1
      i: 1
      i: 10
      i: 1
    }
  }
}
attr {
  key: "use_cudnn_on_gpu"
  value {
    b: true
  }
}
